lib/validation.js

```diff
@@ -318,7 +318,7 @@ module.exports = function (yargs, usage, y18n) {
         conflicting[key] = []
       }
       if (Array.isArray(value)) {
-        conflicting[key].concat(value)
+        conflicting[key] = conflicting[key].concat(value)
```

Comment:
```js
// Kinda sucks having to replace the array object instead of just mutating it,
// plus we have to type conflicting[key] twice
conflicting[key] = conflicting[key].concat(value)

// About the same speed as concat + assign, but we get to keep the same array
// object. Still kinda ugly though
Array.prototype.push.apply(conflicting[key], value)

// Twice as fast, less hacky than push.apply, but even more verbose
for (var i = 0, l = implied[key].length; i < l; i++) {
    implied[key].push(value[i])
}

// Should be similar to above, but slightly cleaner
// I think this is my favourite.
for (var i in value) {
    implied[key].push(i)
}

// I didn't consider forEach() since it is known to be way slower than a for loop
```